HW02_B_Mimic_starter.Rmd

```diff
@@ -1,99 +1,122 @@
-set.seed(42)
-```
-
-Now using the mpg dataset and the corvette dataset, make this graph:
-
-```{r graoh 3 code}
-
-```
-
-There is a trick to getting the model and year to print off together. `paste()` is a useful function for this, also pasting together parts of file names and parts of urls together. 
-
-
-### Graph 4 
-```{r, echo = T}
-data(mpg)
-
-#hint for the coloring, colorbrewer and you can set palette colors and make your graphs colorblind friendly
-library(RColorBrewer)
-display.brewer.all(colorblindFriendly = T) #take a look at the colorblindfriendly options
-```
-
-The above graph lets you see some colobrlind friendly palettes. For the graph below, I used Set2.
-
-
-
-Now using the above mpg dataset, make this graph
-
-```{r graph 4 code}
-
-```
-
-
-
-
+---
+title: "HW02_B_Graph-Mimic"
+author: "YOUR NAME HERE"
+output: github_document
+---
+
+```{r setup, include=FALSE}
+knitr::opts_chunk$set(echo = FALSE)
+```
+
+
+```{r load libraries, echo = TRUE, include=TRUE}
+library("ggplot2")
+library("magrittr") #so I can do some piping
+data("diamonds")
+data("mpg")
+data("iris")
+theme_set(theme_bw()) #I'll give you this one, you can set the theme individually for graphs
+#or you can set the theme for all the graphs using theme_set()
+#theme_bw() is best theme (IMO)
+
+#for graph 3:
+library("ggrepel")
+
+
+```
+
+## HW02 Part B
+
+For this part of the HW, the goal is to try to recreate the graphs I make from scratch. I will only provide the MD, not the actual code I used to create it besides which data I use to create it. The rest will be up to you. 
+
+Try for all 4, but if you are struggling don't worry about it. Try your best for each, if you don't get everything that's what the peer-review is for. :smile:
+
+
+### Graph 1
+```{r, echo = T, include = TRUE}
+data("diamonds")
+ggplot(diamonds, aes(cut, ,fill = clarity))+
+  geom_bar(position = "dodge")+
+  labs(x = "Diamond Cut", y = "Number of Diamonds", title = "My Diamond Collection", subtitle = "Boxplot representing the number of diamonds in my diamond collection by \ntype of cut quality and clarity of diamond") +
+  theme(plot.title = element_text(hjust = 0.5))+
+  annotate("text", x =4, y = 4500, label = "My Best Diamonds,\n of course")+
+  annotate("rect", xmin = 4.5,xmax = 5.5, ymin = 0 ,ymax = 5000, alpha = 0.35)
+  #theme(plot.subtitle = element_text(margin = ))
+#hint think about the *position* the bars are in...
+```
+
+
+Using the diamonds dataset, make this graph:
+```{r graph1 code, echo=FALSE}
+
+
+```
+
+### Graph 2
+```{r, echo = T, include = TRUE}
+data("iris")
+
+```
+
+Using the iris dataset, make this graph:
+```{r graph 2 code, echo=FALSE}
+
+ggplot(iris, aes(Sepal.Length, Petal.Length, shape = Species))+
+  geom_point(aes(color = Species))+
+  facet_wrap(~ Species, scales = "free_y")+
+  geom_smooth(method = lm, color = "black", se = F)
+#can't figure out how to change the order of the facets.
```

Comment:
Your code looks good overall! I had the same issue with not being able to change the order of the facets. I tried implementing a transform, but it didn't work... Maybe someone else in our group succeeded.

Reply:
This worked for me:
facet_wrap(factor(Species,levels=c("versicolor","setosa","virginica"))~.,scales="free_y")